server/server.js

```diff
@@ -10,4 +10,5 @@ server.use(express.static(path.join(__dirname, './public')))
 
 server.use('/', starterRoute)
 
+
```

Comment:
This line doesn't add anything - perhaps remove?

Reply:
sure , deleted.